src/hooks/use-schema.js

```diff
@@ -0,0 +1,14 @@
+import { useContext, useEffect, useState } from 'react';
```

Comment:
Good catch that you don't need to import `React` itself from react- since there's no jsx in this file.